src/test/groovy/org/prebid/server/functional/CacheSpec.groovy

```diff
@@ -41,14 +36,15 @@ class MetricsSpec extends BaseSpec {
         assert metrics["account.${accountId}.prebid_cache.requests.ok" as String] == 1
     }
 
-    @PendingFeature
     def "PBS should update prebid_cache.creative_size.json metric when json creative is received"() {
         given: "Current value of metric prebid_cache.requests.ok"
         def initialValue = getCurrentMetricValue("prebid_cache.requests.ok")
 
-        and: "Default BidRequest"
+        and: "Default BidRequest with cache, targeting"
         def bidRequest = BidRequest.defaultBidRequest
         bidRequest.enableCache()
+        def priceGranularity = new PriceGranularity(ranges: [new Range(max: 20, increment: 0.5)])
+        bidRequest.ext.prebid.targeting = new Targeting(priceGranularity: priceGranularity)
```

Comment:
This can be a part of the `enableCache()` as the caching won't work without targeting.